<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HPE-OT" failures="23" tests="23" name="SKU API Tests" time="36.786" errors="0" timestamp="2025-08-21T21:57:55 CDT">
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSku" time="15.57">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSku: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSku: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSku(SkuApiTest.java:137)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSku -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuDuplicate" time="0.949">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuDuplicate: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuDuplicate: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuDuplicate(SkuApiTest.java:279)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuDuplicate -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuEmptyBody" time="0.92">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuEmptyBody: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuEmptyBody: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuEmptyBody(SkuApiTest.java:331)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuEmptyBody -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuInvalidPrice" time="0.942">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuInvalidPrice: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 60
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuInvalidPrice: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 60

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuInvalidPrice(SkuApiTest.java:299)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuInvalidPrice -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuLongDescription" time="0.923">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuLongDescription: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 550
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuLongDescription: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 550

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuLongDescription(SkuApiTest.java:317)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuLongDescription -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuMissingDescription" time="0.938">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuMissingDescription: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 33
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuMissingDescription: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 33

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuMissingDescription(SkuApiTest.java:233)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuMissingDescription -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuMissingPrice" time="0.931">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuMissingPrice: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 46
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuMissingPrice: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 46

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuMissingPrice(SkuApiTest.java:252)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuMissingPrice -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuMissingSku" time="0.996">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuMissingSku: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 44
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuMissingSku: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 44

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuMissingSku(SkuApiTest.java:214)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuMissingSku -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuWithExtraFields" time="0.906">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuWithExtraFields: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 82
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuWithExtraFields: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 82

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuWithExtraFields(SkuApiTest.java:490)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuWithExtraFields -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testCreateSkuWithInvalidPayload" time="0.912">
    <failure type="java.lang.AssertionError" message="Exception during testCreateSkuWithInvalidPayload: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 38
">
      <![CDATA[java.lang.AssertionError: Exception during testCreateSkuWithInvalidPayload: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 38

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testCreateSkuWithInvalidPayload(SkuApiTest.java:195)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testCreateSkuWithInvalidPayload -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testDeleteSku" time="0.982">
    <failure type="java.lang.AssertionError" message="Exception during testDeleteSku: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/berliner
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testDeleteSku: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/berliner
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testDeleteSku(SkuApiTest.java:174)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testDeleteSku -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testDeleteSkuInvalidIdFormat" time="0.922">
    <failure type="java.lang.AssertionError" message="Exception during testDeleteSkuInvalidIdFormat: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/!!!
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testDeleteSkuInvalidIdFormat: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/!!!
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testDeleteSkuInvalidIdFormat(SkuApiTest.java:429)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testDeleteSkuInvalidIdFormat -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testDeleteSkuNonExistent" time="0.937">
    <failure type="java.lang.AssertionError" message="Exception during testDeleteSkuNonExistent: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/random123
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testDeleteSkuNonExistent: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/random123
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testDeleteSkuNonExistent(SkuApiTest.java:416)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testDeleteSkuNonExistent -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testDeleteThenGetOrUpdate" time="0.936">
    <failure type="java.lang.AssertionError" message="Exception during testDeleteThenGetOrUpdate: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/deleted
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testDeleteThenGetOrUpdate: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → DELETE http://localhost:8080/api/skus/deleted
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testDeleteThenGetOrUpdate(SkuApiTest.java:528)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testDeleteThenGetOrUpdate -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testGetAllSkus" time="0.921">
    <failure type="java.lang.AssertionError" message="Exception during testGetAllSkus: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testGetAllSkus: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testGetAllSkus(SkuApiTest.java:149)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testGetAllSkus -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testGetSkuById" time="0.967">
    <failure type="java.lang.AssertionError" message="Exception during testGetSkuById: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus/berliner
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testGetSkuById: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus/berliner
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testGetSkuById(SkuApiTest.java:162)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testGetSkuById -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testGetSkuByNonExistentId" time="0.965">
    <failure type="java.lang.AssertionError" message="Exception during testGetSkuByNonExistentId: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus/random123
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testGetSkuByNonExistentId: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus/random123
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testGetSkuByNonExistentId(SkuApiTest.java:344)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testGetSkuByNonExistentId -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUnauthorizedAccess" time="0.92">
    <failure type="java.lang.AssertionError" message="Exception during testUnauthorizedAccess: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
">
      <![CDATA[java.lang.AssertionError: Exception during testUnauthorizedAccess: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → GET http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUnauthorizedAccess(SkuApiTest.java:503)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUnauthorizedAccess -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUnsupportedHttpMethodPatch" time="0.937">
    <failure type="java.lang.AssertionError" message="Exception during testUnsupportedHttpMethodPatch: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → PATCH http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61
">
      <![CDATA[java.lang.AssertionError: Exception during testUnsupportedHttpMethodPatch: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → PATCH http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUnsupportedHttpMethodPatch(SkuApiTest.java:469)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUnsupportedHttpMethodPatch -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUnsupportedHttpMethodPut" time="1.057">
    <failure type="java.lang.AssertionError" message="Exception during testUnsupportedHttpMethodPut: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → PUT http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61
">
      <![CDATA[java.lang.AssertionError: Exception during testUnsupportedHttpMethodPut: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → PUT http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUnsupportedHttpMethodPut(SkuApiTest.java:449)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUnsupportedHttpMethodPut -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUpdateSkuInvalidPrice" time="1.008">
    <failure type="java.lang.AssertionError" message="Exception during testUpdateSkuInvalidPrice: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61
">
      <![CDATA[java.lang.AssertionError: Exception during testUpdateSkuInvalidPrice: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 61

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUpdateSkuInvalidPrice(SkuApiTest.java:384)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUpdateSkuInvalidPrice -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUpdateSkuMissingSku" time="1.113">
    <failure type="java.lang.AssertionError" message="Exception during testUpdateSkuMissingSku: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 45
">
      <![CDATA[java.lang.AssertionError: Exception during testUpdateSkuMissingSku: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 45

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUpdateSkuMissingSku(SkuApiTest.java:403)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUpdateSkuMissingSku -->
  <testcase classname="ChallengeAPIPlaywright.Framework.SkuApiTest" name="testUpdateSkuValid" time="0.957">
    <failure type="java.lang.AssertionError" message="Exception during testUpdateSkuValid: Error {
  message=&amp;apos;connect ECONNREFUSED ::1:8080
  name=&amp;apos;Error
  stack=&amp;apos;Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 62
">
      <![CDATA[java.lang.AssertionError: Exception during testUpdateSkuValid: Error {
  message='connect ECONNREFUSED ::1:8080
  name='Error
  stack='Error: connect ECONNREFUSED ::1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
}
Call log:
-   - → POST http://localhost:8080/api/skus
-     - user-agent: Playwright/1.53.1 (x64; windows 10.0) java/17
-     - accept: */*
-     - accept-encoding: gzip,deflate,br
-     - Content-Type: application/json
-     - content-length: 62

at org.testng.Assert.fail(Assert.java:111)
at ChallengeAPIPlaywright.Framework.SkuApiTest.testUpdateSkuValid(SkuApiTest.java:364)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </failure>
  </testcase> <!-- testUpdateSkuValid -->
</testsuite> <!-- SKU API Tests -->
